File: app/src/main/java/com/Phase4/pathways/model/ProjectAdapter.java
package com.Phase4.pathways.model;

import android.view.View;
import android.view.ViewGroup;

import androidx.annotation.NonNull;
import androidx.recyclerview.widget.RecyclerView;

class ProjectAdapterAdapter extends RecyclerView.Adapter<ProjectAdapterAdapter.ProjectViewHolder> {


    @NonNull
    @Override
    public ProjectViewHolder onCreateViewHolder(@NonNull ViewGroup parent, int viewType) {
        return null;
    }

    @Override
    public void onBindViewHolder(@NonNull ProjectViewHolder holder, int position) {

    }

    @Override
    public int getItemCount() {
        return 0;
    }

    static class ProjectViewHolder extends RecyclerView.ViewHolder {


        public ProjectViewHolder(@NonNull View itemView) {
            super(itemView);
        }
    }

}

File: app/src/main/java/com/Phase4/pathways/model/ProjectAdapter.kt
package com.Phase4.pathways.model

import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.widget.ImageView
import android.widget.TextView
import androidx.recyclerview.widget.RecyclerView
import com.Phase4.pathways.R

class ProjectAdapter(val project: List<Project>) : RecyclerView.Adapter<ProjectViewHolder>() {

    private var titles =
        arrayOf("Software Developer", "Data Science", "IT Specialist", "UX Designer")

    private var description = arrayOf(
        "This how you become a software developer",
        "This is how you become a data scienctist",
        "This is how you become a IT specialist",
        "This how you become a UX Designer"
    )


    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): ProjectViewHolder {
        val view = LayoutInflater.from(parent.context).inflate(R.layout.item_project, parent, false)
        return ProjectViewHolder(view)
    }

    override fun onBindViewHolder(holder: ProjectViewHolder, i: Int) {

        holder.textViewTitle.text = titles[i]
        holder.textViewDesc.text = description[i]

    }

    override fun getItemCount(): Int {
        return titles.size
    }
}

class ProjectViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
     var imageView: ImageView = itemView.findViewById(R.id.image_view_project_icon)
     var textViewTitle: TextView = itemView.findViewById(R.id.text_view_project_title)
     var textViewDesc: TextView = itemView.findViewById(R.id.text_view_project_desc)


    fun bindView(project: Project) {

    }


}